racket/src/cs/core/control.ss

```diff
@@ -806,6 +841,8 @@
 (define-record elem+cache (elem cache))
 (define (elem+cache-strip t) (if (elem+cache? t) (elem+cache-elem t) t))
 
+
+;; TODO: 
```

Comment:
I don't think anything is needed here, since `call-with-immediate-continuation` can't look past the start of a future.